ta_services/keymaster/km.c

```diff
@@ -5,6 +5,27 @@
 
 #include "km.h"
 
+bool version_info_set = false;
+uint32_t boot_os_version = 0;
+uint32_t boot_os_patchlevel = 0;
+
+TEE_Result km_configure(uint32_t os_version, uint32_t os_patchlevel)
+{
+	IMSG("os_version = %u", os_version);
+	IMSG("os_patchlevel = %u", os_patchlevel);
+
+	if (!version_info_set) {
+		//https://android.googlesource.com/trusty/app/keymaster/+/994293cc45700fa58512b312c94da0f46d95403e
```

Comment:
Please use the `/* ... */` way of commenting instead of `//`.

Reply:
Interesting comment on the URL too, seems like the patch/version becomes almost useless now.

Reply:
ok